lib/beaker-hostgenerator/hypervisor/vmpooler.rb

```diff
@@ -14,28 +14,18 @@ def global_config()
       end
 
       def generate_node(node_info, base_config, bhg_version)
-        host_config = {}
-        host_config.deep_merge! base_config
-
         # set hypervisor
-        host_config['hypervisor'] = 'vmpooler'
-
-        # generate node definition
-        ostype = node_info['ostype']
-        nodeid = node_info['nodeid']
-        bits = node_info['bits']
-
-        platform = "#{ostype}-#{bits}"
-        name = "#{platform}-#{nodeid}"
```

Comment:
I moved the name generation out of the hypervisor and into the single generator because the name is no longer hypervisor-specific.